conn/physic/units.go

```diff
@@ -154,6 +154,44 @@ func (e ElectricResistance) String() string {
 	return nanoAsString(int64(e)) + "Ω"
 }
 
+// Set sets the ElectricResistance to the value represented by s. Units are to
+// be provided in Ohm/Ohms or Ω with an optional SI prefix (n,p,u,µ,m,k,M,G or
+// T)
+func (f *ElectricResistance) Set(s string) error {
+	v, n, err := valueOfUnitString(s, nano)
+	if err != nil {
+		if e, ok := err.(*parseError); ok {
+			switch e.err {
+			case errOverflowsInt64:
+				return errors.New("maximum value is " + maxElectricResistance.String())
+			case errUnderflowsInt64:
+				return errors.New("minimum value is " + minElectricResistance.String())
+			case errNotANumber:
+				found, _ := containsUnitString(s, "Ohm", "Ohms", "Ω")
+				if found != "" {
+					return errors.New("does not contain number")
+				}
+				return errors.New("does not contain number or unit \"Ohm\"")
+			}
+		}
+		return err
+	}
+
+	switch strings.ToLower(s[n:]) {
+	case "ohm", "ohms", "ω": // ω is lowercase Ω
```

Comment:
Ah! I think we should not support "ω" as this would be confusing. An option is to first check manually, then do the rest:
```
if rest := s[n:]; rest == "Ω" {
  *f = (ElectricResistance)(v)
} else {
  switch strings.Lower(rest) {
  case "ohm", "ohms":
      *f = (ElectricResistance)(v)
  case "":
    ..
  default:
    ..
  }
}
```

Reply:
If == Ω I prefer it it just return right there since the rest of the code has nothing to do with it any more.